server/controllers/userController.js

```diff
@@ -0,0 +1,27 @@
+import validator from 'validator';
+
+import db from '../models';
+
+const { User } = db;
+
+class UserController {
+  /**
+   * Create User Record
+   * @param {object} req
+   * @param {object} res
+   */
+  static async createUser(req, res) {
+    const {
+      firstname, lastname, email, password,
+    } = req.body;
+    const response = await User.create({
+      firstname,
+      lastname,
+      email,
+      password,
```

Comment:
Looks like you did not hash the password. 

Again, looks like you are returning the password in the response. It should not be because of security reasons.

You can use 
`delete response.dataValues.password` to remove the password from the response

Reply:
I actually hashed the password, take a look at my model, using sequelize one can add a function hook on the model so it can be hashed from there, I would remove the password from the response body.